### DISCUSSION TITLE
Jump to the opening window/tab instead of current window

### DISCUSSION THREAD

**Discussion by LintaoAmons** [danielfalk/smart-open.nvim#84] [Ideas]
When the selected item was already show in one of the existing windows, instead of opening it up in current window, jumping to the existing window.

E.g. 
Tab1: win1 / win2 / win3(target.md)
Tab2: win4(current active window)

When smart-open `target.md` in `win4`, it will jump to `Tab1/win3`

**Comment by megalithic** [comment]
love this idea!

**Comment by danielfalk** [comment]
Can this be implemented as a telescope action?  It seems like this is something that other telescope-based file pickers would get the exact same benefit from.
